crypto/x509v3/v3_asid.c

```diff
@@ -830,6 +859,10 @@ static int asid_validate_path_internal(X509_STORE_CTX *ctx,
  */
 int X509v3_asid_validate_path(X509_STORE_CTX *ctx)
 {
+    if (ctx->chain == NULL
```

Comment:
This is just error checking, not assert. We are allowed to return an error if we detect one!